src/main/resources/static/PasswordValidation.js

```diff
@@ -0,0 +1,27 @@
+function validatePassword(){
+    var passwordIn = document.getElementById("password");
+    var passwordCf = document.getElementById("passwordConfirm");
+    if(passwordIn.value != passwordCf.value) {
+        passwordCf.setCustomValidity("Passwords Don't Match");
```

Comment:
why is every word written with capital letter?

Reply:
for Better readability so you know that the passwordCf means password Confirm

Reply:
How is Passwords Don't Match more readable than Passwords don't match ^^

Reply:
ah I thought you meant just 'passwordIn', 'passwordCf'